deployment/sklearn/sklearn-census-income-prediction.ipynb

```diff
@@ -0,0 +1,617 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8lCXP3A__nPN"
+      },
+      "source": [
+        "# Census Income Prediction (scikit-learn)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rjbaCPG4_vjE"
+      },
+      "source": [
+        "## 1. Dependencies"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EFsTgnMe_w9i"
+      },
+      "source": [
+        "This notebook has been tested with **Python 3.8.16** and the following package versions:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YYZpIDPcky8c"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!pip install verta==0.21.1\n",
```

Comment:
Why peg the version to `0.21.1` instead of just `pip install verta` to get the latest?

Reply:
Good question! I could go either way on this.

I think it's helpful to have an indication of "This notebook has been verified to work with version `0.21.1` of our client" (I've seen libraries out there update their package's public API but not their examples, leaving things in a confusing state for learners).

On the other hand, in principle we should always ensure notebooks work with our latest version.

🤔

Reply:
> Why peg the version to 0.21.1 instead of just pip install verta to get the latest?

Because most NBs I used as reference specify Verta's package version.

I think this gives more assurance that the NB will work, and we can change the version number as we need for further NBs. But as @liuverta said, we should ensure NBs work with the latest version as well.

I can easily remove the package version and the NB will probably work for now - but on new package releases it would be interesting to test them again.

Let me know how you guys prefer.

Reply:
Ah ok.  I have only dealt with a limited number of these, so I didn't realize that was the existing pattern.  I'd say leave it pegged at 0.22.1 for now, in that case.  I think we (Verta) need to talk about the broader strategy for maintaining these examples going forward, but that is another conversation for another time.

Reply:
Great. I read the document you sent in Slack, it's a good way to go.
Ping me if I can help you with that :handshake: